### PR TITLE
fix: Rename KodaDot to Koda throughout the app

### LINKED ISSUE #10854: Rename `KodaDot` to `Koda` throughout the app
since

translation files should also be updated to reflect that

### PR DESCRIPTION
**Pull request by rajithaeyee**
## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Needs Design check

- @exezbcz please review

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [x] Closes #10854

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

### CODE REVIEW

File: locales/en.json [hunk 1]
```diff
@@ -617,8 +617,8 @@
         "networkFee": "Network Fee",
         "existentialDeposit": "NFT Existential Deposit",
         "depositTooltip": "A deposit of {0} {1} is required to create an item. Please note, this initial deposit is refundable. ",
-        "kodadotFee": "KodaDot Fee",
-        "kodadotTooltip": "By keeping this option active, you're contributing to initial costs borne by Kodadot for storing your JPEGs on your behalf. You may change this preference anytime in the settings. ",
+        "KodaFee": "Koda Fee",
+        "KodaTooltip": "By keeping this option active, you're contributing to initial costs borne by Koda for storing your JPEGs on your behalf. You may change this preference anytime in the settings. ",
```
Comment (hassnian) [inline locales/en.json:621]: 1. translation key is being updated without updating it's usage
2. key should be camelCase

File: locales/hi.json [hunk 2]
```diff
@@ -1053,8 +1053,8 @@
         "depositTooltip": "एक ITEM बनाने के लिए {0} {1} का जमा किया जाना चाहिए। कृपया ध्यान दें, यह प्रारंभिक जमा वापसी के योग्य है।",
         "existentialDeposit": "NFT अस्तित्वी संरक्षण जमा",
         "intoCollection": "संग्रह में",
-        "kodadotFee": "KodaDot शुल्क",
-        "kodadotTooltip": "इस विकल्प को सक्रिय रखकर आप अपने JPEG कोडाओं को धारित करने के लिए Kodadot द्वारा उठाए गए प्रारंभिक खर्चों में सहायक हो रहे हैं। आप अपनी पसंद को Settings में कभी भी बदल सकते हैं।",
+        "KodaFee": "Koda शुल्क",
+        "KodaTooltip": "इस विकल्प को सक्रिय रखकर आप अपने JPEG कोडाओं को धारित करने के लिए Koda द्वारा उठाए गए प्रारंभिक खर्चों में सहायक हो रहे हैं। आप अपनी पसंद को Settings में कभी भी बदल सकते हैं।",
```
Comment (hassnian) [inline locales/hi.json:1057]: same

**Review by hassnian — CHANGES_REQUESTED** [review]

File: locales/en.json (hunk 1 above)
Comment (rajithaeyee) [reply locales/en.json]: @hassnian crap my apologies, corrected

File: locales/hi.json (hunk 2 above)
Comment (rajithaeyee) [reply locales/hi.json]: corrected.

File: locales/en.json [hunk 3]
```diff
@@ -1522,7 +1522,7 @@
     "everyThursday": "Every Thursday",
     "addToCal": "Add To Calendar",
     "createACalendarEvent": "Create A Calendar Event",
-    "kodadotWeeklyGenerativeDrop": "Kodadot weekly generative drop",
+    "KodaWeeklyGenerativeDrop": "Koda weekly generative drop",
```
Comment (hassnian) [inline locales/en.json:1525]: same here

**Review by hassnian — CHANGES_REQUESTED** [review]
@rajithaeyee pls check all your changes and fix any invalid key

File: locales/en.json (hunk 3 above)
Comment (rajithaeyee) [reply locales/en.json]: @hassnian Double checked everything, and looks good to me except one place, I kept **https://hello.kodadot.xyz/multi-chain/migrator** as it is, as i saw there is no valid web app up and running for **https://hello.koda.art/multi-chain/migrator**.

**Comment by vikiival** [comment]
cc @hassnian

**Review by vikiival — APPROVED** [review]

**Review by hassnian — APPROVED** [review]

**Pull request merged**